components/tag-list/tag-list.js

```diff
@@ -68,11 +69,16 @@ class TagList extends LocalizeCoreElement(ArrowKeysMixin(LitElement)) {
 				visibility: hidden;
 			}
 			.d2l-tag-list-clear-button {
+				position: absolute;
```

Comment:
This was added because of the clear button taking up space invisibly in list (I can remove if this change is already being made elsewhere - just thought I'd throw it in while I was here)

Reply:
Looks great!  My PR will have some ugly conflicts - I will just get your changes and reapply mine on top to use the mixin.